assets/src/dashboard/app/views/editorSettings/adManagement/index.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useCallback } from 'react';
+import { __, TranslateWithMarkup } from '@web-stories-wp/i18n';
+import { trackClick } from '@web-stories-wp/tracking';
+/**
+ * Internal dependencies
+ */
+import { THEME_CONSTANTS } from '../../../../../design-system';
+import { AD_NETWORK_TYPE } from '../../../../constants';
+import {
+  InlineLink,
+  MultilineForm,
+  SettingHeading,
+  TextInputHelperText,
+} from '../components';
+import AdNetworkSettings from './adNetwork';
+import GoogleAdManagerSettings from './googleAdManager';
+import GoogleAdSenseSettings from './googleAdSense';
```

Comment:
Kinda waffled about nesting these vs keeping them all at `/editorSettings` - they are individual forms but are definitely reliant on one and other for when to show up so I opted to nest them but am absolutely game to move them back if other people find that more readable.

Reply:
I like the way these are set up